generic_hook_test.go

```diff
@@ -53,6 +53,11 @@ func TestGetLoggingLevels(t *testing.T) {
 			minimumLevel: logrus.TraceLevel,
 			levels:       []logrus.Level{logrus.PanicLevel, logrus.FatalLevel, logrus.ErrorLevel, logrus.WarnLevel, logrus.InfoLevel, logrus.DebugLevel, logrus.TraceLevel},
 		},
+		{
+			name:         "over",
+			minimumLevel: 7,
```

Comment:
I really don't like unnamed levels. Actually, I hate them with a passion
- I can never remember if 0 is high or low. It's not even the same in every programming languages
- It makes code completely unreadable, I couldn't figure out what was happening in gotemplate. You knew what was happening because you wrote it.
- Don't we have enough with TRACE? TRACE is supposed to be almost unreadable
- We return a logrus.Level with them from them but it's only a broken shell. Every function you call from these objects will be bad because logrus doesn't support them. Also, AcceptedLevels doesn't show them. This is terrible for discoverability. How am I to know that I can set the log level to 10 and again I'm a user, is 10 a high logging level or is 10 the lowest?

Reply:
I understand you, but what you saw is not additional name for level, it's a test to see what's happen when we set the level higher than expected. The previous behaviour was totally wrong, I just fixed it.